anchore_engine/services/catalog/__init__.py

```diff
@@ -2512,6 +2524,49 @@ def garbage_collect_imports():
                 logger.exception("Error deleting image, may retry on next cycle")
 
 
+def list_account_import_ops(account: str, session=None, limit: int = 1000):
+    """
+    Return list of import operations for the given account, but only the first limit count to keep memory use reasonable.
+
+    :param account:
+    :param session:
+    :param limit:
+    :return: list of ImageImportOperations objects
+    """
+
```

Comment:
May be add a check for `session` before using

Reply:
Yep, good catch.